README.rst

```diff
@@ -81,6 +85,14 @@ The following configuration options are supported:
 
   - List of images to associate with a queue name so that they can be referenced by name when using testflinger reserve
 
+- **global_timeout**:
+
+  - Maximum global timeout (in seconds) a job is allowed to specify for this device agent. The job will timeout during the provision or test phase if it takes longer than the requested global_timeout to run. (Default 4 hours)
+
+- **output_timeout**:
+
+  - Maximum output timeout (in seconds) a job is allowed to specify for this device agent. The job will timeout if there has been no output in the test phase for longer than the requested output_timeout. (Default 15 min.)
```

Comment:
I'm not an English expert and of course not a native speaker, but "maximum timeout" sounds kind of weird to me. Can we have a minimum timeout?
Probably it's better to say "time (in seconds) during which a just is allowed ...". I believe you have better ideas, but you got my point :)

Reply:
I think it is trying to say that the job itself written by the client/user can set a timeout it wants applied, but the maximum length of timeout that the client can set as enforced by facility owner is global|output_timeout. I think these personas do need to be described in the documentation ultimately because I don't think it's obvious for someone coming fresh to the project

Reply:
@jocave so, do you think that this description is fine for now?

Reply:
I think it's possibly ok for this Readme, but I don't think we should think there is enough documentation for here for some to understand testflinger setup as a whole. Hopefully with a good tutorial, explanation docs etc we should be able to do that.